README.md

```diff
@@ -1,3 +1,138 @@
+
+# How to Use
+
+PlayTools is shipped with PlayCover, and is installed together with the game. You do not need to install PlayTools manually.
+
+PlayTools provides in-game menus mostly for key mapping setup. Other features are accessible through the PlayCover settings menu.
+
+Localization is handled in [Weblate](https://hosted.weblate.org/projects/playcover/).
+
+# How it Works
+
+PlayTools runs alongside of the game. During IPA installation, PlayTools is inserted into the game's executable. When the game is launched, PlayTools will be launched together.
+
+PlayTools uses swizzle techniques to replace framework methods and system calls with the versions provided by PlayTools. During game play, PlayTools intercepts game input events and translates them to the corresponding touch events and feed them to the game.
+
+# How to Build
+
+PlayTools is built using Xcode.
+
+## Production Build
+In release builds, the building script of PlayCover will automatically fetch the latest version of PlayTools from the official repository and build it. Generally, you do not need to build PlayTools manually.
+
+If you want to build PlayCover with a specific version of PlayTools, mostly for testing purposes, you can change the [Cartfile](https://github.com/PlayCover/PlayCover/blob/develop/Cartfile) of PlayCover to point to the specific version of PlayTools. You can specify which branch/tag of which repository to build from.
+
+## Development Build
+PlayTools can also be built separately. This is useful when you want to modify the source code.
+
+To do so, 
+
+1. Clone the PlayTools repository.
+
+1. Open the PlayTools project in Xcode.
+
+1. Set the development team in the Xcode project settings. Both `PlayTools` and `AKInterface` targets need to be set. You may need to create a development team first.
+
+1. Build PlayTools towards iOS platform. This will create a `PlayTools.framework` in the build directory.
+
+2. Transform the target platform to Mac Catalyst. Run this script in the build directory:
+```bash
+#!/bin/sh
+echo "Converting to maccatalyst"
+vtool \
+	-set-build-version maccatalyst 11.0 14.0 \
+	-replace -output \
+	"PlayTools.framework/PlayTools" \
+	"PlayTools.framework/PlayTools"
+
+echo "Codesigning PlayTools"
+codesign -fs- "PlayTools.framework/PlayTools"
+```
+
+2. Copy the `PlayTools.framework` into the PlayCover App, and replace the old one.
+```bash
+cp "PlayTools.framework" "PlayCover.app/Contents/Frameworks/PlayTools.framework"
+```
+
+3. Relaunch PlayCover.
+
+### Temporary Build
+
+If you are debugging and testing your own code, relaunching PlayCover every time you make a change is a bit annoying. 
+
+In most cases, you can just copy the `PlayTools.framework/PlayTools` to `~/Library/Frameworks/PlayTools.framework/` instead of the whole `PlayTools.framework` directory, and avoid relaunching PlayCover:
+
+```bash
+cp "PlayTools.framework/PlayTools" "~/Library/Frameworks/PlayTools.framework/"
```

Comment:
Does PlayTools need to be codesigned when using a temporary build (I think I've ran into an issue with signing before, that might have been fixed)?

Reply:
It needs codesign. That converting script needs to be run in either cases. And that script contains the codesign step.